src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -4,30 +4,24 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
-using System.Linq;
-using System.Text;
 
 namespace System.Diagnostics
 {
     internal static partial class ProcessManager
     {
         /// <summary>Gets the IDs of all processes on the current machine.</summary>
-        public static int[] GetProcessIds()
-        {
-            return EnumerateProcessIds().ToArray();
-        }
+        public static int[] GetProcessIds() => new List<int>(EnumerateProcessIds()).ToArray();
```

Comment:
This allocates an extra array and copies from one to the other?

Reply:
In general no more than it was previously.